includes/WPGraphQLContentBlocks.php

```diff
@@ -119,6 +110,38 @@ static function () {
 	 */
 	public function actions(): void {
 		add_action( 'graphql_register_types', [ $this, 'init_block_editor_registry' ] );
+		$this->register_wpgraphql_update_filters();
+	}
+
+	/**
+	 * Register filters related to WPGraphQL version compatibility.
+	 *
+	 * @since 0.0.1
+	 */
+	public function register_wpgraphql_update_filters(): void {
+		add_filter( 'graphql_get_dependents', [ $this, 'add_as_wpgraphql_dependent' ], 10, 2 );
+
+		add_filter( 'wpgraphql_enable_untested_autoupdates', function ($allow, $release_type) {
```

Comment:
This filter is for whether _WPGraphQL_ should autoupdate for any plugin, not just this one.

But also, untested autoupdates are allowed by default for non _major_ versions: https://github.com/wp-graphql/wp-graphql/blob/98457dc6a3412e34ba2f6cbeef848982904ab7a8/src/Admin/Updates/UpdateChecker.php#L352